src/components/sections/talk-announcement/talk-announcement.scss

```
@@ -32,11 +34,18 @@
 
 .talk-announcement-image {
     margin-top: 16px;
-    width: 100%;
-    height: auto;
-    border-radius: 20px;
+    object-fit: cover;
+    border-radius: 50%;
     background-color: var(--cl-neutral);
 
+    @include mixins.media(xs, md) {
+        margin-top: 0;
+        width: 100%;
+        height: auto;
+        object-fit: cover;
+        border-radius: 52px;
+    }
```

Repeating default styles

```suggestion
    object-fit: cover;
    border-radius: 50%;
    background-color: var(--cl-neutral);

    @include mixins.media(xs, md) {
        margin-top: 0;
        width: 100%;
        height: auto;
        border-radius: 52px;
    }
```